src/System.Private.Xml/tests/XmlSerializer/System.Xml.XmlSerializer.Tests.csproj

```diff
@@ -11,15 +11,9 @@
     <AssemblyName>System.Xml.XmlSerializer.Tests</AssemblyName>
     <ProjectGuid>{4050F1D1-1DD2-4B48-A17B-E3F90DD18C4B}</ProjectGuid>
     <DefineConstants Condition="'$(TargetGroup)' == 'netcore50' OR '$(TargetGroup)' == 'netcore50aot'">$(DefineConstants);NET_NATIVE</DefineConstants>
-    <!--
-        Until we get first class support for NS1.7 and Netcoreapp1.1
-        we need to hard code $(TestTFM) and $(TestNugetTargetMoniker)
-        in the project file.
-      -->
-      <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
-      <TestTFM>netcoreapp1.1</TestTFM>
-      <KeepAllProjectReferences>true</KeepAllProjectReferences>
-    </PropertyGroup>
+    <NugetTargetMoniker>.NETStandard,Version=v1.7</NugetTargetMoniker>
+    <KeepAllProjectReferences>true</KeepAllProjectReferences>
```

Comment:
We need an issue to remove KeepAllProjectReferences as that ends up opt'ing it out of testing against our packages during our official builds which can mean we have packaging bugs that go unnoticed in our test passes.